routes/ProductionRequest.ts

```diff
@@ -0,0 +1,121 @@
+import { Request, Response, NextFunction } from 'express';
+import Joi from '@hapi/joi';
+import logger from '../config/logger';
+import GovDeliveryService from '../services/GovDeliveryService';
+import { DevPortalError } from '../models/DevPortalError';
+import {validateApiList} from './DeveloperApplication';
+
+export const productionSchema = Joi.object().keys({
+  primaryContact: Joi.object({
+    firstName: Joi.string().required(),
+    lastName: Joi.string().required(),
+    email: Joi.string().required(),
+  }).required(),
+  secondaryContact: Joi.object({
+    firstName: Joi.string().required(),
+    lastName: Joi.string().required(),
+    email: Joi.string().required(),
+  }).required(),
+  organization: Joi.string().required(),
+  appName: Joi.string().required(),
+  appDescription: Joi.string().required(),
+  statusUpdateEmails: Joi.array().items(Joi.string()).required(),
+  valueProvided: Joi.string().required(),
+  businessModel: Joi.string(),
+  policyDocuments: Joi.array().items(Joi.string()).required(),
+  phoneNumber: Joi.string().required(),
+  apis: Joi.custom(validateApiList).required(),
+  monitizedVeteranInformation: Joi.boolean().required(),
+  monitizationExplanation: Joi.string(),
+  veteranFacing: Joi.boolean(),
+  website: Joi.string(),
+  signUpLink: Joi.string(),
+  supportLink: Joi.string(),
+  platforms: Joi.array().items(Joi.string()),
+  veteranFacingDescription: Joi.string(),
+  vasiSystemName: Joi.string(),
+  credentialStorage: Joi.string().required(),
+  storePIIOrPHI: Joi.boolean().required(),
+  storageMethod: Joi.string(),
+  safeguards: Joi.string(),
+  breachManagementProcess: Joi.string(),
+  vulnerabilityManagement: Joi.string(),
+  exposeHealthInformationToThirdParties: Joi.boolean(),
+  thirdPartyHealthInfoDescription: Joi.string(),
+  scopesAccessRequested: Joi.array().items(Joi.string()),
+  distrubitingAPIKeysToCustomers: Joi.boolean(),
+  namingConvention: Joi.string(),
+  centralizedBackendLog: Joi.string(),
+  listedOnMyHealthApplication: Joi.boolean(),
+}).options({ abortEarly: false });
```

Comment:
missing links to terms of service pages?

Reply:
held inside `policyDocuments` https://github.com/department-of-veterans-affairs/developer-portal-backend/blob/4b4ffa06a3b2983cd18be75e1ad1b369b75ba8e7/routes/ProductionRequest.ts#L25